src/Application/Adapters/Laravel/Commands/InsightsCommand.php

```diff
@@ -50,7 +52,14 @@ public function handle(): int
         $analyseCommand = $container->get(AnalyseCommand::class);
 
         $output = (new Reflection($this->output))->get('output');
-        return $analyseCommand->__invoke($this->input, $output);
+
+        $result = $analyseCommand->__invoke($this->input, $output);
+
+        if ($output instanceof ConsoleOutputInterface) {
+            $output->getErrorOutput()->writeln('✨ See something that needs to be improved? <options=bold>Create an issue</> or send us a <options=bold>pull request</>: <fg=cyan;options=bold>https://github.com/nunomaduro/phpinsights</>');
+        }
```

Comment:
Why does this line exists two times? Would it be possible to eliminate that duplication?

Reply:
:thinking: 

I removed it from AnalyseCommand to put it in InvocableCommand, as I add a new FixCommand. 

Then, when I tested on a Laravel Application, I realized we don't use the InvocableCommand, but directly the AnalyseCommand.

Maybe I should change the Command call for laravel.

Reply:
Hmm, else I don't think it's that a big problem, we can just leave it for now 👍